OneMore/AddIn.cs

```diff
@@ -142,7 +145,7 @@ private void CatchUnhandledException(object sender, UnhandledExceptionEventArgs
 		/// Gets the thread culture for use in subsequent threads; used primarily for 
 		/// debugging when explicitly setting the culture in the AddIn() constructor
 		/// </summary>
-		public static CultureInfo Culture { get; private set; } = CultureInfo.GetCultureInfo("en");
+		public static CultureInfo Culture { get; private set; } = getCulture();
```

Comment:
I don't understand this change. All you've done is set the Culture property before the constructor runs, rather than from inside the constructor. In the original code, `Culture` was set to "en" as a default but then overwritten by the constructor to the user's preferred language from OneMore settings. You perhaps streamlined it a microsecond made no real changes here.